src/Libraries/RevitNodes/Filter/OverrideGraphicSettings.cs

```diff
@@ -50,18 +44,24 @@ internal OverrideGraphicSettings(Autodesk.Revit.DB.OverrideGraphicSettings inter
         /// <param name="projectionFillPattern">Projection fill pattern</param>
         /// <param name="cutLinePattern">Cut line pattern</param>
         /// <param name="projectionLinePattern">Projection line pattern</param>
-        /// <returns>OverrideGraphicSettings</returns>
+        /// <param name="transparency">Transparency as integer between 1-100.</param>
+        /// <param name="detailLevel">Detail Level setting, ex: Coarse, Fine etc.</param>
+        /// <param name="halftone">Halftone. True = halftone.</param>
+        /// <returns name="overrides">Override Graphic Settings</returns>
         public static OverrideGraphicSettings ByProperties(
```

Comment:
This node WAS NOT WORKING to begin with. I stated this before. There is no way for user to obtain unwrapped `Autodesk.Revit.DB.FillPatternElement` which is what this input was declared as. All Python based nodes automatically wrap elements and there is no access to this from a CodeBlock...basically it was a dead end. You could not provide these inputs for this node. By changing the type to `Revit.Elements.Element` it allows you to query Fill Patterns using Python or the DropDown nodes that I added in this PR.